tachyon/math/polynomials/univariate/univariate_polynomial.h

```diff
@@ -119,13 +122,25 @@ class UnivariatePolynomial final
     return coefficients_.Evaluate(point);
   }
 
+  template <typename Container>
+  constexpr static Field EvaluateVanishingPolyByRootsSerial(
+      const Container& roots, const Field& point) {
+    Field ret = Field::One();
+    for (size_t i = 0; i < roots.size(); ++i) {
+      ret *= (point - roots[i]);
+    }
+    return ret;
```

Comment:
```suggestion
    return std::accumulate(roots.begin(), roots.end(), Field::One(),
                           [point](Field& acc, const Field& root) {
                             return acc *= (point - root);
                           });
```
You can just reuse the accumulate right?

Reply:
Yep, makes sense. I'll take that.